doc/development/squid.md

```diff
@@ -71,182 +85,218 @@ Parameters:
 * web3Dto - Web3 client wrapped in a DTO object to avoid coupling with specific web3 specific libraries
 * providerDto - Ocean Provider client wrapped in a DTO object
 
-This method returns an instance of the **OceanManager** class. It allows to get access to the other Ocean core methods
+This method returns an instance of the **Ocean** class. It allows to get access to the other Ocean core methods
 
-## Ocean
 
-Interface with core Ocean functions
+In Python 
+Instantiate an ocean object
+
+
+```python
+   ocean = Ocean(keeper_url = '', provider_url = '', [web3=web3Object] ...)
+```
 
-### Balance
+## Ocean
 
+Interface with core Ocean functions
 
 #### Functions
 
+* **getAccounts** - Returns all avalible accounts loaded via a wallet, or by Web3.
```

Comment:
IMO this function is not a Must Have. 
Not having an accounts registry, it's difficult to retrieve all the accounts interacting with the system. I'm not sure if it's necessary or recommended either.

Reply:
i think it is, here is an excerpt from my PR:

> re added getAccounts because we need them. We don't have an actor registry but we still have accounts (that are coming from the blockchain node). We need access to those informations. Maybe accounts is the wrong term. Shall we call it wallets maybe? Because thats how we use it right now. We can topple up wallets (requestTokens), we can list wallets (getAccounts).

Reply:
Certainly agree it's a must have. Missed that one, I was going with the Python style of removing the 'get' from the property name, so ```getAccounts``` became ```accounts```

To make things easier we can just return the account address, similar to Web3

Reply:
I've been stuck on wallets for a few weeks, trying to solve this one. Possibly copy [Web3.js](https://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html)?

Where we load/create a wallet using a Wallet class, and then only pass the account address from the wallet to the squid library.

So the ```getAccounts``` in the squid library  is just a helper for Web3.getAccounts()

Reply:
For clarify myself, is the expected behavior of this function, to get a list in your local node of all the existing accounts? Like the Parity Wallet client is doing?
For example, in squid-js will load this info from Metamask, in squid-py from the python wallet (ie. pywallet)